src/commands/repos/UpdateRepos.ts

```diff
@@ -139,6 +123,45 @@ export class UpdateRepos extends AbstractReposCommand {
         Global.isVerbose() && console.log('successfully updated', repoName);
     }
 
+    private async checkoutTagOrBranch(repo: Repository, repoName: string, commit: string, branch: string, tag: string): Promise<void> {
+        let tagToCheckout: string = tag;
+        if (branch.startsWith('release/')) {
```

Comment:
to avoid checking for tags when having a commit (and a release branch), maybe
```suggestion
        if (!commit && branch.startsWith('release/')) {
```